MTM-Frontend/src/pages/Auth/Register.tsx

```diff
@@ -33,6 +33,14 @@ interface FormValues {
   affiliation?: string;
 }
 
+interface organization {
+  id: number;
+  name: string;
+  type: string;
+}
+
+const serverBaseUrl = "http://localhost:3001";
```

Comment:
Let's refactor this into the frontend .env file as an environment variable, since later we will be connecting to deployed server